ci-operator/config/openshift/jenkins-openshift-login-plugin/openshift-jenkins-openshift-login-plugin-master.yaml

```diff
@@ -22,6 +22,10 @@ images:
         source_path: /go/src/github.com/openshift/jenkins-openshift-login-plugin/.
   to: jenkins
 
+promotion:
```

Comment:
Given you bumped the `4.1` to `4.3` in the other two files, maybe you should do it here too?